src/backend/libs/efiling-cso-starter/src/main/java/ca/bc/gov/open/jag/efilingcsostarter/CsoSubmissionServiceImpl.java

```diff
@@ -43,24 +44,27 @@ public CsoSubmissionServiceImpl(FilingFacadeBean filingFacadeBean,
     @Override
     public BigDecimal submitFilingPackage(
             AccountDetails accountDetails,
-            EfilingService service,
+            FilingPackage efilingPackage,
             EfilingFilingPackage filingPackage,
             boolean isRushedProcessing,
             EfilingPaymentService paymentService) {
 
-        if(accountDetails == null) throw new IllegalArgumentException("Account Details are required");
-        if(service == null) throw new IllegalArgumentException("Service is required.");
+        if(accountDetails == null) throw new IllegalArgumentException("Account Details is required");
+        if(accountDetails.getClientId() == null) throw new IllegalArgumentException("Service id is required.");
```

Comment:
is serviceId same as clientId??
```suggestion
        if(accountDetails.getClientId() == null) throw new IllegalArgumentException("Client id is required.");
```